src/modules/airspeed_selector/airspeed_selector_params.c

```diff
@@ -69,25 +69,128 @@ PARAM_DEFINE_INT32(ARSP_TAS_GATE, 3);
  * @unit SD
  * @group Airspeed Validator
  */
-PARAM_DEFINE_INT32(ARSP_BETA_GATE, 1);
+PARAM_DEFINE_INT32(ASPD_BETA_GATE, 1);
 
 /**
  * Automatic airspeed scale estimation on
  *
- * Turns the automatic airspeed scale (scale from IAS to CAS/EAS) on or off. It is recommended level (keeping altitude) while performing the estimation. Set to 1 to start estimation (best when already flying). Set to 0 to end scale estimation. The estimated scale is then saved in the ARSP_ARSP_SCALE parameter.
+ * Turns the automatic airspeed scale (scale from IAS to CAS/EAS) on or off. It is recommended to fly level
+ * altitude while performing the estimation. Set to 1 to start estimation (best when already flying).
+ * Set to 0 to end scale estimation. The estimated scale is then saved using the ASPD_SCALE parameter.
  *
  * @boolean
  * @group Airspeed Validator
  */
-PARAM_DEFINE_INT32(ARSP_SCALE_EST, 0);
+PARAM_DEFINE_INT32(ASPD_SCALE_EST, 0);
 
 /**
  * Airspeed scale (scale from IAS to CAS/EAS)
  *
- * Scale can either be entered manually, or estimated in-flight by setting ARSP_SCALE_EST to 1.
+ * Scale can either be entered manually, or estimated in-flight by setting ASPD_SCALE_EST to 1.
  *
  * @min 0.5
  * @max 1.5
  * @group Airspeed Validator
  */
-PARAM_DEFINE_FLOAT(ARSP_ARSP_SCALE, 1.0f);
+PARAM_DEFINE_FLOAT(ASPD_SCALE, 1.0f);
+
+/**
+ * Index or primary airspeed measurement source
+ *
+ * @value -1 Disabled
+ * @value 0 Groundspeed minus windspeed
+ * @value 1 First airspeed sensor
+ * @value 2 Second airspeed sensor
+ * @value 3 Third airspeed sensor
+ *
+ * @reboot_required true
+ * @group Airspeed Validator
+ */
+PARAM_DEFINE_INT32(ASPD_PRIMARY, 1);
```

Comment:
We'll probably need to find a better way to do this later. We can't rely on the ordering being maintained in all cases.

Reply:
Agree. As we currently though anyway just support 1 sensor it's not yet that critical.